src/ui/popup/components/link.tsx

```diff
@@ -0,0 +1,22 @@
+import {m} from 'malevic';
+
+interface LinkProps {
+    cls: string;
+    url: string;
+}
+
+function openTab(url: string) {
+    return () => chrome.tabs.create({url});
+}
+
+export default function Link({cls, url}: LinkProps, ...children) {
+    return (
+        <a
+            class={cls}
+            onclick={openTab(url)}
+            rel="noopener noreferrer"
```

Comment:
This can be removed if `chrome.tabs` is being used. As it won't have any affect.

Reply:
True, good point.